projects/worker-basic/README.md

```diff
@@ -0,0 +1,18 @@
+# worker-basic
+
+Simple example of a 3rd party Asset Compute worker. It generates a rendition by simply [copying](https://github.com/adobe/asset-compute-example-workers/blob/master/projects/worker-basic/worker.js#L19) the source file.
```

Comment:
Relative URL should work too, then it doesn't have to be touched when folder or repos are renamed: `worker.js#L19`